### ISSUE TITLE
Hide people already in circles

### ISSUE THREAD

**Issue by mbriney** [voloko/facebook-circles#9]
A great addition to this would be the ability to hide anyone who is already in a circle.

**Comment by nandonakis** [comment]
Yes, I was going to request the same thing. Many people have many friends that are not in lists. 
At least on initial use, the primary function would be drag and drop them into a primary list. Hiding people already allocated from the list of friends on the top effectively makes it a "move" drag and drop operation rather than a "copy" operation.

The only proviso with doing this though is that a lot of people are going to be in stupid default lists like city name...and you want to be able to ignore those groups from the facilty above.
